### PR TITLE
I18n support for cukes. Use i18n for labels login, register and 'forgot password'.

### PR DESCRIPTION
**Pull request by machisuji**
Related to [WP 5555 (multi-provider login forms)](https://www.openproject.org/work_packages/5555) where labels had been changed, which then had to be reverted since they broke a ton of tests.

This PR restores those label changes.

Following the merge of this PR changes to the cukes of to the following plugins are needed (PRs linked):
- [openproject-mobile_otp](https://github.com/finnlabs/openproject-mobile_otp/pull/21)
- [openproject-landing_page](https://github.com/finnlabs/openproject-landing_page/pull/23)
- [openproject-self_registration_by_mail](https://github.com/finnlabs/openproject-self_registration_by_mail/pull/15)

### CODE REVIEW

**Comment by mfrister** [comment]
Thanks for fixing this!

Is there a way to do this without patching Capybara? I generally don't like patches as they tend to break in the future and are not visible (someone trying to find out why t:label works won't find this one easily).

Is it an option to add extra step definitions that work with I18n or change the existing ones to support both? We could e.g. create a function that checks for the `t:` prefix and translates a string if that's present. We could then use this function in all step definitions where we need it.

For specs, I think doing this explicitly is even better, although more work now.

**Comment by myabc** [comment]
Agree with @meeee about monkey-patching. Speaking of which, #1271 updates Capybara.

File: features/session/user_session.feature [hunk 1]
```diff
@@ -36,7 +36,7 @@ Feature: User session
     Then I should be on the login page
     When I fill in "bob" for "username" within "#login-form"
     And I fill in "adminADMIN!" for "password" within "#login-form"
-    And I click on "Login" within "#login-form"
+    And I click on "t:button_login" within "#login-form"
```
Comment (linki) [inline features/session/user_session.feature]: i would prefer removing the prefix: `And I click on ":button_login"`

Comment (machisuji) [inline features/session/user_session.feature]: That would give the impression of a symbol, though, which it isn't. Example: `t:account.delete`.

**Comment by machisuji** [comment]
@myabc @meeee What about now? Is [this](https://github.com/opf/openproject/commit/e7e88f8f36285a0e53d10384f6657de8a82db876) more to your liking?

**Comment by myabc** [comment]
:+1:

**Comment by machisuji** [comment]
@meeee I [agree](https://github.com/opf/openproject/commit/103bdc638eaabbbb0663ea7a3ee0ef3ad881429e).

**Comment by machisuji** [comment]
@myabc @meeee If you don't have any more qualms about this feel free to merge it along with the linked (see PR description) related plugin PRs, which together make both the myproject and org config green with the changed labels.

**Pull request merged**